nss-day-cohort-48/fun-website-1-faith-s-children
# My East Tn code so far

This is my basic page I have started. I added main page name, hike name, images, and hike specs.

## Steps to  test

1. Pull this branch
2. Open in chrome
3.  Verify there is a header, hike sections with name, image, and specs.
